### DISCUSSION TITLE
Is it posible to use with wired controllers?

### DISCUSSION THREAD

**Discussion by Ronbinn** [darthcloud/BlueRetro#833] [Q&A]
Hi

I was thinking about doing a universal arcade stick for my current consoles (pce, famicom, ps2 and gamecube). My idea was to wire the stick directly to an arduino/ripico/esp32/whatever and wire a db-25 on the other side. Then create db-25 adapters for each console.

Now i've found this awesome project, which does exactly what I wanted... but with bluetooth controllers. So my question is: is it possible to modify the code and use wired controllers instead?

**Comment by darthcloud** [comment]
Well everything is possible if you put time into it.

But that's definitely not something I will work on. This project is about Bluetooth only.
